lib/manifest.js

```diff
@@ -290,6 +290,18 @@ internals.manifest = [
         list: true,
         useFilename: internals.passthruOn('name')
     },
+    {
+        place: 'events',
+        method: 'event',
+        signature: ['events'],
```

Comment:
I think you can nix the `signature` for this one.  By default haute will pass the contents of a file as the first argument.  In this case it might be the difference between `{ events: { name: 'my-event' } }` and `{ name: 'my-event' }`.  The latter is cleaner, as we can get away with it in this case.